indexer/lib/typings.d.ts

```diff
@@ -14,41 +14,3 @@ export interface ProviderInfo {
 
 export type ProviderToInfoMap = Map<string, ProviderInfo>;
 
-/**
-lastAdCID --> [ ] -\
-               ↓    |
-              ...   | entries announced after we started the current walk
-               ↓    |
-              [ ] -/
-               ↓
-     head --> [ ] -\
-               ↓    |
-              ...   | entries visited in this walk
-               ↓    |
-              [ ] -/
-               ↓
-     tail --> [ ] -\
-               ↓    |
-              ...   | entries NOT visited yet
-               ↓    |
-              [ ] -/
-               ↓
- lastHead --> [ ] -\
-               ↓    |
-              ...   | entries visited in the previous walks
-               ↓    |
-              [ ] -/
-               ↓
-             (null)
- */
-export interface WalkerState {
-  head?: string;
-  tail?: string;
-  lastHead?: string;
-  status: string;
-}
-
-export type ProviderToWalkerStateMap = Map<string, WalkerState>
-
-export type PiecePayloadCIDs = string[];
-
```

Comment:
This is a follow-up for https://github.com/filecoin-station/piece-indexer/pull/18/files#diff-6c702830fadfd5340905905bcfc38b05b06f0c9ebfb92df7afb61e4c4bbeb8eb where I forgot to remove the definition of types re-exported from the shared package.

Reply:
You can find the other definition here:
https://github.com/filecoin-station/piece-indexer/blob/main/repository/lib/typings.d.ts